### ISSUE TITLE
Python: ModuleNotFoundError: No module named 'samples.service_settings'

### ISSUE THREAD

**Issue by shahramjoon** [microsoft/semantic-kernel#9726]
in my azure environment, i have a notebook.

my
pip show semantic-kernel

command returns

Name: semantic-kernel
Version: 1.15.0
Summary: Semantic Kernel Python SDK
Home-page:
Author:
Author-email: Microsoft [SK-Support@microsoft.com](mailto:SK-Support@microsoft.com)
License:
Location: /anaconda/envs/azureml_py38/lib/python3.10/site-packages
Requires: aiohttp, azure-identity, cloudevents, defusedxml, jinja2, nest-asyncio, numpy, openai, openapi_core, opentelemetry-api, opentelemetry-sdk, prance, pybars4, pydantic, pydantic-settings
Required-by:

i am getting the error

from samples.service_settings import ServiceSettings

Cell In[15], line 3
      1 from services import Service
----> 3 from samples.service_settings import ServiceSettings

ModuleNotFoundError: No module named 'samples.service_settings'

from running Users
semantic-kernel/python/samples/getting_started/00-getting-started.ipynb

**Comment by moonbox3** [comment]
Hello @shahramjoon, it looks like you have Semantic Kernel installed via Anaconda/AzureML? Can you please check that your Kernel is pointing to the correct location where Semantic Kernel is installed?

**Comment by moonbox3** [comment]
We haven't heard from you in a while. We will close this issue. If you need further help, please re-open or file a new issue, thank you.
